app/assets/stylesheets/views/_header.css.scss

```diff
@@ -192,6 +192,29 @@ $elementSpacing: lines(0.5);
   margin-left: lines(0.5);
 }
 
+.header-wide-logo-text,
+.header-square-logo-text {
+  margin-top: 0.625rem;
+  margin-bottom: 0.625rem;
+  line-height: 2.5rem;
+  height: 2.5rem;
+  font-size: em(28);
+  letter-spacing: -0.03em;
+  text-transform: uppercase;
+  font-weight: 800;
+}
+
+.header-wide-logo-text {
+  @include ellipsis;
+  width: em(150);
```

Comment:
This and the line below needs to be rems, since this element's dimensions can't follow font-size.